src/TodoList.js

```diff
@@ -16,13 +16,18 @@ const todoList = [
       title: "Start assignment"
     }
   ];
-const TodoList = (props) => {
+
```

Comment:
I belive the assignment requires you to delete this `todoList` variable on line 5

Reply:
correct, thank you